xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountChannelsRequestParams.java

```diff
@@ -47,22 +44,15 @@
   Optional<Address> destinationAccount();
 
   /**
-   * A 20-byte hex string for the ledger version to use.
+   * Specifies the ledger version to request. A ledger version can be specified by ledger hash,
+   * numerical ledger index, or a shortcut value.
    *
-   * @return An optionally-present {@link Hash256}.
+   * @return A {@link LedgerSpecifier} specifying the ledger version to request.
    */
-  @JsonProperty("ledger_hash")
-  Optional<Hash256> ledgerHash();
-
-  /**
-   * The ledger index of the ledger to use, or a shortcut string to choose a ledger automatically.
-   *
-   * @return A {@link LedgerIndex}.  Defaults to {@link LedgerIndex#CURRENT}.
-   */
-  @JsonProperty("ledger_index")
   @Value.Default
-  default LedgerIndex ledgerIndex() {
-    return LedgerIndex.CURRENT;
+  @JsonUnwrapped
+  default LedgerSpecifier ledgerSpecifier() {
```

Comment:
Is there a risk of breaking clients making this change as it currently stands? Seems like this new method is replacing the old one?

Reply:
Yes this will require a 3.0 release. This is fine because we also have some other breaking changes coming down the pipeline

Reply:
Resolving this comment now that this PR is non-breaking.